describe/EnumType.ts

```diff
@@ -1,5 +1,11 @@
 import { NamingDescription, describeNaming } from "./Naming.js";
 
+export function isEnumTypeDescription(
+  type: EnumTypeDescription
+): type is EnumTypeDescription {
```

Comment:
Same thing, this probably happens in all your type guards, so if you intend to export these for general consumption I'd grep for `function is` and change all their type signatures to make them useful as type guards.